docker/docs/Dockerfile

```
@@ -9,21 +9,31 @@ FROM ${PARENT_IMAGE}:${IMAGE_VERSION} AS pecandocs
 
 RUN apt-get update \
     && apt-get install -y --no-install-recommends pandoc \
-    && Rscript \
-        -e 'repos <- c(getOption("repos"),' \
-        -e '           sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
-        -e 'remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
-        -e 'remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
-        -e 'remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
+    && Rscript -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos"))); \
+                   remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos); \
+                   remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos); \
+                   remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos); \
+                   install.packages("pkgdown")' \
```

Passing all of this as one `-e` puts the line continuations in the R command (where it won't understand them) rather than interpreted by the shell.
```suggestion
    && Rscript \
        -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos")))' \
        -e 'remotes::install_version("rmarkdown", ">= 2.19", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
        -e 'remotes::install_version("knitr", ">= 1.42", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
        -e 'remotes::install_version("bookdown", ">= 0.31", dependencies = TRUE, upgrade = FALSE, repos = repos)' \
        -e 'install.packages("pkgdown")' \
```